ShippoTesting/APIResourceTest.cs

```diff
@@ -1,23 +1,23 @@
 ﻿using NUnit.Framework;
 using System;
-using Shippo;
 using System.Net;
 
+using Shippo;
+
+
 namespace ShippoTesting {
-	[TestFixture()]
+	[TestFixture]
```

Comment:
did you mean to remove the parens here? do we not require instantiation of the fixture?